### PR TITLE
Fix vulnerability problem of helm-operator

### PR DESCRIPTION
**Pull request by hlts2**
### WHAT

I fixed the vulnerability problem of vald-helm-operator.
The following is an main fix.
- make the helm-operator binary work with distroless. 

### WHY

According to `Snyk`, there was a vulnerability error due to the [base image](https://github.com/operator-framework/operator-sdk/blob/master/images/helm-operator/Dockerfile#L20) of helm-operator.
Ref: https://security.snyk.io/vuln/SNYK-DEBIAN11-GLIBC-2340914

### Environment:

- Go Version: 1.17.5
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.13.8

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [x] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dockers/operator/helm/Dockerfile [hunk 1]
```diff
@@ -72,10 +74,21 @@ RUN make helm/schema/vald
...
 RUN cp -r charts /charts
 
-FROM quay.io/operator-framework/helm-operator:${OPERATOR_SDK_VERSION}
+FROM quay.io/operator-framework/helm-operator:${OPERATOR_SDK_VERSION} AS operator
+
+FROM ${DISTROLESS_IMAGE}:${DISTROLESS_IMAGE_TAG}
 LABEL maintainer "${MAINTAINER}"
 
-COPY --from=builder /tmp/watches.yaml ${HOME}/watches.yaml
+ENV HOME=/opt/helm
```
Comment (hlts2) [inline dockers/operator/helm/Dockerfile:82]: Ref: https://github.com/operator-framework/operator-sdk/blob/master/images/helm-operator/Dockerfile#L22

**Comment by hlts2** [comment]
I deployed vald-helm-operator and valdrelease.
The vald cluster was working fine.

```
$ kubectl get pods
NAME                                  READY   STATUS    RESTARTS   AGE
vald-helm-operator-68cb4c55cb-ggkkf   1/1     Running   0          10m
vald-helm-operator-68cb4c55cb-mxglc   1/1     Running   0          10m
vald-agent-ngt-0                      1/1     Running   0          10m
vald-agent-ngt-1                      1/1     Running   0          10m
vald-agent-ngt-2                      1/1     Running   0          10m
vald-agent-ngt-4                      1/1     Running   0          10m
vald-agent-ngt-3                      1/1     Running   0          10m
vald-discoverer-779f7f788-hvs8z       1/1     Running   0          10m
vald-lb-gateway-649cbb99fd-zxfnt      1/1     Running   0          9m45s
vald-lb-gateway-649cbb99fd-vr7ss      1/1     Running   0          9m45s
vald-lb-gateway-649cbb99fd-9vtq2      1/1     Running   0          10m
vald-manager-index-768dcb9469-cqm9q   1/1     Running   0          10m
```

**Comment by hlts2** [comment]
The following is an security check result using `snyk` command in local.
There was no vulnerability problem.

```
$ snyk container test vdaas/vald-helm-operator:pr-1535

Testing vdaas/vald-helm-operator:pr-1535...

Organization:      hlts2
Package manager:   deb
Project name:      docker-image|vdaas/vald-helm-operator
Docker image:      vdaas/vald-helm-operator:pr-1535
Platform:          linux/amd64
Licenses:          enabled

✔ Tested 3 dependencies for known issues, no vulnerable paths found.
```

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**